core/lib/petsciiconv.c

```diff
@@ -73,13 +73,13 @@ static unsigned char ascii2petscii[128] = {
 };
 
 static unsigned int i;
-static unsigned char *ptr;
+static char *ptr;
 
```

Comment:
Oh, and looking at this file again. i and ptr do not need to be static variable*, but can be put into a local scope as automatic variables. Saves a few bytes of global data memory.

* Do not confuse with the static key-word. :)